Vault-Deployment/manifests/loki-alerting-rules/loki-rules.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: loki-alerting-rules
+  namespace: loki
+data:
+  rules.yaml: |
+    groups:
+      - name: vault-requests
+        rules:
+          - record: vault:requests:rate1m
+            expr: |
+              sum(rate({application="vault"} |~ "\"type\":\"request\"" [1m]))
```

Comment:
Note, this is logql, not to be confused with the promql in the alerting rule below. Also, the structure of the YAML is different (similar at first with the groups and rules, but different)!